src/bin/huak/commands/install.rs

```diff
@@ -13,7 +13,7 @@ pub fn run() -> CliResult<()> {
         Err(e) => return Err(CliError::new(e, ExitCode::FAILURE)),
     };
 
-    if let Err(e) = ops::install::install_project_dependencies(&project) {
+    if let Err(e) = ops::install::install_project_dependencies(&project, &[]) {
```

Comment:
So from what I understand, this change makes it so that `huak install` doesn't install any optional deps? Do we want that?

Reply:
Looking at how poetry does it, poetry by default installs everything. I'm actually not sure I like that. Optional deps are *optional*. So *opting into* them I think would make sense. See below.

Reply:
I wasn't sure myself if by default all or no opt dep should be installed.